layouts/partials/html-head.hbs

```diff
@@ -18,4 +18,7 @@
     <link rel="alternate" href="/{{site.locale}}/tsc-minutes.xml" title="Node.js TSC meeting minutes" type="application/rss+xml">
     <link rel="stylesheet" href="/{{site.locale}}/styles.css" media="all">
     <link rel="stylesheet" href="/static/css/prism-tomorrow.css" media="all">
+
+    <script src="//use.typekit.net/mse5tqx.js"></script>
+    <script>try{Typekit.load();}catch(e){}</script>
```

Comment:
It should only be embedded if the file size is < 2k, otherwise two requests are more performant than embedding.
You can also load an optimized version for the user's browser from Google Fonts:
https://www.google.com/fonts#QuickUsePlace:quickUse

For mobile use, it would be best to use the font-loading API and caching the fonts to localStorage to avoid the text being invisible until the web-font has finally loaded.